src/auth/auth.ts

```diff
@@ -291,7 +292,7 @@ class AuthService {
 		return signMessage;
 	}
 
-	public async AddressLogin (address: string, signature: string, wallet: Wallet): Promise<IAuthResponse> {
+	public async AddressLogin (address: string, signature: string, wallet: Wallet, multisigAddress?:string): Promise<IAuthResponse> {

```

Comment:
we don't need another param as `multisigAddress` here. we can pass it straight to `address`